docs/book/how-to/stack-deployment/deploy-a-cloud-stack.md

```diff
@@ -0,0 +1,129 @@
+requires you to first deploy the necessary pieces of infrastructure and then 
+define them as stack components in ZenML with proper authentication.
+
+Especially in a remote setting, this process can be challenging and 
+time-consuming, and it may create multi-faceted problems. This is why we 
+implemented a feature, that allows you to **deploy the necessary pieces of 
+infrastructure on your selected cloud provider and get you started on remote 
+stack with a single click**.
+
+{% hint style="info" %}
+If you have the required infrastructure pieces already deployed on your cloud, 
+you can also use [the stack wizard to seamlessly register your stack](register-a-cloud-stack.md).
+{% endhint %}
+
+## How to use the 1-click deployment tool?
+
+To access the 1-click deployment tool, you can either use the dashboard or 
+the CLI:
+
+{% tabs %}
+{% tab title="Dashboard" %}
+
+In order to create a remote stack over the dashboard go to the stacks page 
+on the dashboard and click "+ New Stack".
+
+![The new stacks page](../../.gitbook/assets/register_stack_button.png)
+
+Since we will be deploying it from scratch, select "New Infrastructure" on the
+next page:
+
+![Options for registering a stack](../../.gitbook/assets/register_stack_page.png)
+
+{% hint style="warning" %}
+Currently, the 1-click deployment only works on AWS. We are working on 
+bringing support to GCP and Azure as well. Stay in touch for further updates.
+{% endhint %}
+
+![Choosing a cloud provider](../../.gitbook/assets/deploy_stack_selection.png)
+
+### AWS
+
+If you choose `aws` as your provider, you will see a page where you will have 
+to select a region and a name for your new stack:
+
+![Configuring the new stack](../../.gitbook/assets/deploy_stack_aws.png)
+
+Once the configuration is finished, you will see a deployment page:
+
+![Deploying the new stack](../../.gitbook/assets/deploy_stack_aws_2.png)
+
+During this process, you  will be redirected you to a Cloud Formation page 
+on AWS. 
+
+![Cloudformation page](../../.gitbook/assets/deploy_stack_aws_cloudformation_intro.png)
+
+
+You will have to log in to your AWS account, review and confirm the 
+pre-filled configuration and create the stack.
+
+![Finalizing the new stack](../../.gitbook/assets/deploy_stack_aws_cloudformation.png)
+
+{% endtab %}
+{% tab title="CLI" %}
+
+In order to create a remote stack over the CLI you can use the following 
+command:
+
+```shell
+zenml stack deploy -p aws
+```
+
+{% hint style="warning" %}
+Currently, the 1-click deployment only works on AWS. We are working on 
+bringing support to GCP and Azure as well. Stay in touch for further updates.
+{% endhint %}
+
+### AWS 
+
+If you choose `aws` as your provider, this command will redirect you to a 
+Cloud Formation page on AWS: 
+
+![Cloudformation page](../../.gitbook/assets/deploy_stack_aws_cloudformation_intro.png)
+
+You will have to log in to your AWS account, review and confirm the 
```

Comment:
also you will need permissions to deploy the AWS cloudformation script

Reply:
done.